src/Performance/Converters/CalendarYearReturnsConverter.ts

```diff
@@ -0,0 +1,101 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Mateusz Bernacik
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import MorningstarConverter from '../../Shared/MorningstarConverter';
+import type { PerformanceConverterOptions } from '../PerformanceOptions';
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class CalendarYearReturnsConverter extends MorningstarConverter {
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+
+    public constructor (
+        options?: PerformanceConverterOptions
+    ) {
+        super(options);
+    }
+
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+
+    public override parse (
+        options: PerformanceConverterOptions,
+        hasMultiple?: boolean
```

Comment:
Doesn't work with multiple portfolios (only one is parsed):

<img width="554" height="237" alt="Image" src="https://github.com/user-attachments/assets/5b027b0e-1ce1-4741-8f08-1408d4d618bd" />